### DISCUSSION TITLE
What is disable_scroll_handling ?

### DISCUSSION THREAD

**Discussion by Linh2196** [sveltejs/kit#7153] [General]
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'disable_scroll_handling') at navigation.js?v=94b1394a:16:11

**Comment by david-plugge** [comment]
When does the error occur? I got the same error when creating hooks.client.ts and creating and calling a function inside. Pretty weird

**Reply by Linh2196** [reply]
When I run vite dev, it writes log at the first time I access this site, but this log disappears when I reload
